drivers/mspi/Kconfig.nrfe

```diff
@@ -25,4 +25,23 @@ config MSPI_NRFE_INIT_PRIORITY
 	  SDP MSPI initialization depends on IPC initialization
 	  which is done at the same init level and has init priority equal to 46.
 
+choice MSPI_NRFE_IPC_TRANSFER_MODE
+	prompt "Ipc data transfer mode"
+	default MSPI_NRFE_IPC_NO_COPY
+	help
+	  Select the data transfer mode for IPC.
+
+config MSPI_NRFE_IPC_COPY
+	bool "IPC copy mode"
+	help
+	  Data is passed through IPC by copy.
+
+config MSPI_NRFE_IPC_NO_COPY
```

Comment:
I would add a line `depends on SOC_NRF54L15` or `depends on SOC_SERIES_NRF54LX` or `depends on !SOC_SERIES_NRF54HX` to make sure that it is not selected for platforms, whose cores do not have access to all memory.
Same goes for FLPR application.